.github/workflows/audit.yml

```diff
@@ -0,0 +1,18 @@
+name: Security audit
+
+on:
+  push:
+    paths: 
+      - '**/Cargo.toml'
+      - '**/Cargo.lock'
```

Comment:
This doesn't make sense as we don't add Cargo.lock in this repo, right?

Reply:
Yes, we didn't. But just in case.